archive/100-theorems-list/70_perfect_numbers.lean

```diff
@@ -61,7 +61,7 @@ theorem even_two_pow_mul_mersenne_of_prime (k : ℕ) (pr : (mersenne (k + 1)).pr
   even ((2 ^ k) * mersenne (k + 1)) :=
 by simp [ne_zero_of_prime_mersenne k pr] with parity_simps
 
-lemma eq_pow_two_mul_odd {n : ℕ} (hpos : 0 < n) :
+lemma eq_sq_mul_odd {n : ℕ} (hpos : 0 < n) :
   ∃ (k m : ℕ), n = 2 ^ k * m ∧ ¬ even m :=
```

Comment:
This one is about `2 ^ k`; maybe it should be `two_pow` instead of `pow_two`, but it shouldn't be `sq`.

Reply:
You're right, it should be `two_pow`.